### PR TITLE
doc: change apt install line to a working command line

### PR DESCRIPTION
**Pull request by buxx**
The backend doc is not working about apt install lines. This PR suggests a working example.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] ~~The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution~~
- [x] ~~Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)~~
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] ~~Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.~~

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by Haskaris** [comment]
I would suggest to merge this PR in [misc/2912__refactoring_documentation](https://github.com/tracim/tracim/tree/misc/2912__refactoring_documentation). However, we will need to find what we will do about dependabot before merging the PR #5627.
Maybe we should separate the dependabot PR from the documentation PR, so we can merge the documentation update.

File: backend/README.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ On Debian Bullseye (11) with sudo:
 
 ```shell
 sudo apt update
-sudo apt install system_packages/debian/build_backend_packages.list
-sudo apt install system_packages/debian/run_backend_packages.list
+xargs sudo apt-get install < system_packages/debian/build_backend_packages.list
+xargs sudo apt install < system_packages/debian/run_backend_packages.list
```
Comment (PhilippeAccorsi) [inline backend/README.md:14]: Why not purpose same syntax for two line ?

**Review by PhilippeAccorsi — COMMENTED** [review]
quesions

File: backend/README.md (hunk 1 above)
Comment (buxx) [reply backend/README.md:14]: Because I'm a dumb. haha, I fix it.

**Comment by PhilippeAccorsi** [comment]
Command line working good.

**Comment by Millefeuille42** [comment]
Done

**Pull request merged**